packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuInput.tsx

```diff
@@ -71,6 +108,8 @@ export const DropdownMenuInput = forwardRef<
       onShiftTab,
       onTab,
       rightComponent,
+      isPhoneInput = false,
```

Comment:
This component doesn't have to know about phone related stuff, it's also generic, we should find a way to avoid this.